lib/macho/view.rb

```diff
@@ -0,0 +1,23 @@
+module MachO
```

Comment:
The indentation in this file is weirdly off. Let's stick to two spaces per level of indentation. :wink: